src/status_im/bottom_sheet/core.cljs

```diff
@@ -22,3 +22,25 @@
   [{:keys [db]}]
   {:hide-bottom-sheet nil
    :db (assoc db :bottom-sheet/show? false)})
+
+(fx/defn show-bottom-sheet-redesign
```

Comment:
i guess you could keep old bottom sheet , just use `new-ui-eneabled` in bottom sheet itself if you need to change UI of bottom sheet